Games/types/Mafia/roles/Village/Mortician.js

```diff
@@ -0,0 +1,11 @@
+const Role = require("../../Role");
+
+module.exports = class Mortician extends Role {
+
+    constructor(player, data) {
+        super("Mortician", player, data);
+        this.alignment = "Village";
+        this.cards = ["VillageCore", "WinWithVillage", "RoleLearnerDead"];

```

Comment:
you can use a meetingMod to modify the "role learner" meeting target to be dead

you can check hooker code